pylint/config/option_manager_mixin.py

```diff
@@ -271,6 +271,14 @@ def read_config_file(self, config_file=None, verbose=None):
 
         use_config_file = config_file and os.path.exists(config_file)
         if use_config_file:
+            mod_name = config_file
+            self.current_name = mod_name
+            try:
+                self.set_current_module(mod_name)
+            except AttributeError:
+                # 'OptionsManagerMixIn' object has no attribute 'set_current_module'
+                # Ie: the class is not used as a mixin
+                pass
```

Comment:
Do we actually use this without initialising `PyLinter`?

Reply:
It seems to be only in ``test_can_read_toml_env_variable`` now (I remembered 4/5 tests being affected), we might want to change the test itself.

Reply:
I think that might be better indeed.

Reply:
This will be fixed in #5284